lib/cib/cib_utils.c

```diff
@@ -328,9 +328,10 @@ cib_perform_op(const char *op, int call_options, cib_op_t * fn, gboolean is_quer
             CRM_CHECK(rc == pcmk_ok, goto done);
 
             pcmk__output_set_log_level(out, LOG_TRACE);
-            pcmk__xml_show_changes(out, scratch);
-            out->finish(out, CRM_EX_OK, true, NULL);
+            rc = pcmk__xml_show_changes(out, scratch);
+            out->finish(out, pcmk_rc2exitc(rc), true, NULL);
```

Comment:
pcmk_rc_no_output -> CRM_EX_IOERR, we probably want to remap that to ok (and anywhere else we'll be using rc2exitc)

Reply:
That mapping is questionable since `pcmk_rc_no_output` is a perfectly normal consequence of `out->is_quiet() == true`. I'm sure there are error conditions that can cause `pcmk_rc_no_output` too.

Reply:
No, I don't remember why we would have done that.  Change it and see what breaks, I guess?

Reply:
It was done when the new code was first added, and neither the commit message nor the PR (#2008) comment on it, so I'm guessing it wasn't chosen for any particular reason. Let's just map it to CRM_EX_OK in pcmk_rc2exitc() and run the regression tests.

Reply:
`cts-regression` passes with the mapping to `CRM_EX_OK`